### PR TITLE
CN Update for 41.42

### PR DESCRIPTION
**Pull request by zlq4863947**

### CODE REVIEW

File: CN/ItemName_CN.txt [hunk 1]
```diff
@@ -272,10 +272,10 @@ ItemName_CN = {
...
     ItemName_Base.CleaningLiquid = "清洁液",
     ItemName_Base.CleaningLiquid2 = "清洁液",
-    ItemName_Base.ClosedUmbrellaBlack = "雨伞（收起）",
-    ItemName_Base.ClosedUmbrellaBlue = "雨伞（收起）",
-    ItemName_Base.ClosedUmbrellaRed = "雨伞（收起）",
-    ItemName_Base.ClosedUmbrellaWhite = "雨伞（收起）",
+    ItemName_Base.ClosedUmbrellaBlack = "黑色雨伞（收起）",
+    ItemName_Base.ClosedUmbrellaBlue = "蓝色雨伞（收起）",
+    ItemName_Base.ClosedUmbrellaRed = "红色雨伞（收起）",
+    ItemName_Base.ClosedUmbrellaWhite = "白色雨伞（收起）",
```
Comment (Mishasama) [inline CN/ItemName_CN.txt:278]: 原文根本没有颜色之分

File: CN/ItemName_CN.txt [hunk 2]
```diff
@@ -1355,14 +1355,14 @@ ItemName_CN = {
     ItemName_Radio.RadioRed = "收音机（高科技）",
     ItemName_Radio.RadioTransmitter = "无线电发射器",
     ItemName_Radio.ScannerModule = "扫描仪模块",
-    ItemName_Radio.TvAntique = "古董电视机",
-    ItemName_Radio.TvBlack = "电视机（廉价技术）",
-    ItemName_Radio.TvWideScreen = "电视机（高科技）",
+    ItemName_Radio.TvAntique = "复古电视机",
+    ItemName_Radio.TvBlack = "显像管电视机",
+    ItemName_Radio.TvWideScreen = "宽屏电视机",
```
Comment (Mishasama) [inline CN/ItemName_CN.txt:1360]: 完全跟原意不一致

File: CN/ItemName_CN.txt [hunk 3]
```diff
@@ -1355,14 +1355,14 @@ ItemName_CN = {
...
     ItemName_Radio.WalkieTalkie1 = "玩具对讲机",
     ItemName_Radio.WalkieTalkie2 = "Valu科技对讲机",
     ItemName_Radio.WalkieTalkie3 = "专业对讲机",
     ItemName_Radio.WalkieTalkie4 = "战术对讲机",
-    ItemName_Radio.WalkieTalkie5 = "美军对讲机",
+    ItemName_Radio.WalkieTalkie5 = "军用对讲机",
```
Comment (Mishasama) [inline CN/ItemName_CN.txt:1365]: 与原文不符

**Review by Mishasama — CHANGES_REQUESTED** [review]
请勿脱离原意进行**“原创”**翻译

File: CN/ItemName_CN.txt (hunk 1 above)
Comment (Alexa-Hong) [reply CN/ItemName_CN.txt:278]: ClosedUmbrella**White**
ClosedUmbrella**Red**
ClosedUmbrella**Blue**
ClosedUmbrella**Black**

File: CN/ItemName_CN.txt (hunk 2 above)
Comment (Alexa-Hong) [reply CN/ItemName_CN.txt:1360]: Tv**Antique**
Tv**WideScreen**

File: CN/ItemName_CN.txt (hunk 1 above)
Comment (HansKaffee) [reply CN/ItemName_CN.txt:278]: 原文是没有颜色之分，但是这样做并不会影响玩家对物品的理解，还能方便玩家分辨颜色。

File: CN/ItemName_CN.txt (hunk 2 above)
Comment (HansKaffee) [reply CN/ItemName_CN.txt:1360]: ValuTech翻译成廉价技术就和原文一致了？

File: CN/ItemName_CN.txt (hunk 3 above)
Comment (zlq4863947) [reply CN/ItemName_CN.txt:1365]: 游戏中有类似军用词汇，所以使用军用更符合游戏设定。
image

File: CN/ItemName_CN.txt (hunk 1 above)
Comment (zlq4863947) [reply CN/ItemName_CN.txt:278]: 我们不喜欢你的独裁翻译，这个翻译是我们汉化组内共同探讨的结果。
要么加群，一起翻译，要么别捣乱

Comment (zhaotaq) [reply CN/ItemName_CN.txt:278]: 已测试，新翻译可以解决，“玩家找不到想要的伞”的问题。
image
image

**Comment by nsKpz** [comment]
Can someone summarize the discussions for me?

**Comment by zlq4863947** [comment]
@nsKpz 
The content submitted now is the result of a joint discussion among 5 people in our Chinese translation team. Please merge.
After the merger, we will announce to Chinese players on these two websites: 
- [blibli](https://www.bilibili.com/)
- [tieba](https://tieba.baidu.com/f?kw=%E5%83%B5%E5%B0%B8%E6%AF%81%E7%81%AD%E5%B7%A5%E7%A8%8B)

Similar to the public address after the last merger:
https://www.bilibili.com/read/cv7516134
https://tieba.baidu.com/p/6931027044?pid=134687771289&red_tag=3013153447#134687771289

If most players return some words that are not accurate enough, the revised content will be submitted in the next PR

**Comment by nsKpz** [comment]
I'm more interested in the discussions that already happened and would like to know the context of them before merging.

**Comment by Alexa-Hong** [comment]
> I'm more interested in the discussions that already happened and would like to know the context of them before merging.

1. 
ItemName_Base.ClosedUmbrellaBlack = "黑色雨伞（收起）",
    ItemName_Base.ClosedUmbrellaBlue = "蓝色雨伞（收起）",
    ItemName_Base.ClosedUmbrellaRed = "红色雨伞（收起）",
    ItemName_Base.ClosedUmbrellaWhite = "白色雨伞（收起）",
“黑色”“蓝色”“红色”“白色”are all colors
@zlq4863947 adds these prefixes so that players can identify the color of the umbrella
Misha believes since the english translation of them do not include the idea of color so Chinese translation should ignore the color part too.

2.
 ItemName_Radio.TvAntique = "复古电视机",
 ItemName_Radio.TvBlack = "显像管电视机",
 ItemName_Radio.TvWideScreen = "宽屏电视机",

 ItemName_Radio.TvAntique = "复古电视机", should be fine

but they have different ideas about ItemName_Radio.TvBlack = "显像管电视机", and  ItemName_Radio.TvWideScreen = "宽屏电视机",
zlq4863947 changes "ValuTech Television"(English translation) from  "电视机（廉价技术）"(Misha's CN translation) to  "显像管电视机"(Zlq's CN translation), "电视机（廉价技术）" means “TV（cheap technology）” in English while  "显像管电视机" means "cathode-ray tube TV" in English.

3. ItemName_Radio.WalkieTalkie5 = "美军对讲机",
    ItemName_Radio.WalkieTalkie5 = "军用对讲机",
“对讲机” means WalkieTalkie
Misha prefer "美军”， which measn "US army”, she suggests translation should loyal to the original text
and zlq prefer "军用" , which means "military”， he changed it because there are other words that use "军用" as well, like “军用头盔”（Military Helmet），"军用沙漠迷彩夹克"(Military Desert Camo Jacket)

**Comment by Alexa-Hong** [comment]
and to be honest, they do not come to an agreement.
This may go on and on and on if they can't find a way working together.

**Comment by nsKpz** [comment]
Thank you @Alexa-Hong 

1.
Regarding Umbrellas and their colours, we have other languages that explicitly state the colour, too. This makes sense to me, and we've considered adding the name to the English original, too. Even if not, I don't see how it would be a valid reason not to change it to something that benefits players more than the original translation. If anything, it'd be a reason for us to adjust the original name as well. This is what this Github is all about.

I have a question regarding these, though. When a player sorts their inventory, players usually prefer items of a different colour to still appear together in the list. 
I noticed that the Chinese name for the umbrellas has the status (collapsed, opened) in brackets for that reason, too. Would it be better to have in the format of "Umbrella - Red (closed)" then? 

I.e. a colour prefix would cause umbrellas of different colours to not appear together when the inventory is sorted by name.

So as an example, would the following work in Chinese?

Umbrella - Black (Collapsed)
雨伞 - 黑色（折叠）

______________________

2. 
If our fake-brand (ValuTech) doesn't have a place in a translation, I am alright with simply calling the item what it is. We do have other items as well as story elements that will mention the name, though, so if there can be a way to have the name still in there while also calling the item something better, that would be preferable. 

The name CRT TV might be a bit odd, as our game takes place at a time where there were no other kinds of TV devices, so nobody would have referred to them as such in the game world, if you know what I mean? 
A more general name for TV might be better?
______________________

3.
I have no real preference either way, so I'm letting you guys come to a consensus if Chinese players would prefer "US Army" or "Military".

**Comment by zlq4863947** [comment]
@Alexa-Hong @nsKpz 
Thank you very much for your suggestions. 
Our translation team has just been established. 
It takes some time to coordinate and discuss, please give us some time and submit the updated content later

**Comment by zlq4863947** [comment]
@nsKpz 

Here are the results of our discussion:
1. We totally agree with you, This way players can classify umbrellas better , and the umbrella-black (foldable) has been tested in the game without problems.



2. We have changed the brand name of the TV.
3. We have consulted many Chinese players, and they all think the word "military" is better .

If there are no other questions, please merge our translated text

**Pull request merged**